### PR TITLE
Preserve app id#229

### PR DESCRIPTION
**Pull request by jperata**
Set app id from the config file if exists and no application ID was provided

### CODE REVIEW

File: lib/client/bst-alexa.ts [hunk 1]
```diff
@@ -68,17 +68,19 @@ export class BSTAlexa {
...
         }
 
-        if (this.sampleUtterancesFile === undefined || this.sampleUtterancesFile === null) {
+        if (!this.sampleUtterancesFile) {
             this.sampleUtterancesFile =  BSTAlexa.DefaultSampleUtterancesLocation;
         }
 
         this._alexa = new Alexa();
-        if (this.applicationID !== undefined && this.applicationID !== null) {
+        if (this.applicationID) {
```
Comment (chris-ramon) [inline lib/client/bst-alexa.ts]: Should not be `Global.config().updateApplicationID(...)` always be call? - perhaps something simpler, like:


```typescript
if (!this.applicationID) {
  this.applicationID = Global.config().applicationID();
}
Global.config().updateApplicationID(this.applicationID);
```

**Comment by chris-ramon** [comment]
Just left a single comment, otherwise LGTM :+1:

File: lib/client/bst-alexa.ts (hunk 1 above)
Comment (jperata) [inline lib/client/bst-alexa.ts]: This set the application id in the machine configuration file, if there's no id present we shouldn't save an empty value.

Comment (chris-ramon) [inline lib/client/bst-alexa.ts]: alright, sounds good to me 👍

**Pull request merged**